pkg/util/validation/notifications_limit_flag_test.go

```diff
@@ -68,7 +68,7 @@ func TestNotificationsLimitMapYaml(t *testing.T) {
 
 	require.NoError(t, testStruct.Flag.Set("{\"email\": 500 }"))
 	expected := []byte(`flag:
-  email: 500
+    email: 500
```

Comment:
is whitespace change required? (wondering if users will need to do anything)

Reply:
It's just that `yaml.Marshal` that is used in this test now encodes with default 4 spaces indentation.

I could change that by using below:
```go
	var buf bytes.Buffer
	enc := yaml.NewEncoder(&buf)
	enc.SetIndent(2)
	err := enc.Encode(testStruct)
	actual := buf.Bytes()
```

But since it's a test string-comparing yaml, I decided to just fix the string. It doesn't affect user experience, any indentation is supperted when unmarshaling.